TaskManagement/project/views.py

```diff
@@ -126,6 +120,24 @@ def get(self, request):
     )
 
         return JsonResponse({"form": html_form})
+
+
+class SearchDateProjectView(View):
+
+    def get(self, request, *args, **kwargs):
+        start = self.request.GET.get('start')
+        end = self.request.GET.get('end')
+        projectlist = get_projects(self.request.user)
+        try:
+            projectlist = projectlist.filter(Q(start_date__icontains__gte=start) & Q(end_date__icontains__lte=end))
+            proj_list = {
+                "search_projectlist": projectlist,
+                "error_message": 'No such data found'
```

Comment:
it will always send error message but you need to send error message only if there is no data